superset/db_engine_specs.py

```diff
@@ -686,6 +805,50 @@ def fetch_result_sets(cls, db, datasource_type, force=False):
         return BaseEngineSpec.fetch_result_sets(
             db, datasource_type, force=force)
 
+    @staticmethod
+    def upload_csv(form, table):
+        "uploads a cv file and creates a superset table in the right databse"
```

Comment:
many (4?) typos and wrong capitalization

Reply:
Fixed!